content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+This notion of structure is akin to Saussure's notion of linguistic structure, which in turn inspired the structuralist tradition in social science:
+on one side, Giddens recalls, in producing an utterance we rely on incorporated syntactic rules;
+on the other side, the production of a meaningful utterance contributes to maintaining language as a structured totality.
+
+<!--
+
+To someone coming from mathematics, such a discussion can be seen as stating the obvious: in a network, the parts' identities are defined by their position respective to the rest of the network, and the totality is made of the identities of the parts.
+So necessarily any definition separating parts and totality will be recursive.
+But the discussion does bear on the distinction between culture and psychology, which exists in academic disciplines.
+
+-->
+
+<!--
+
+Giddens, on the structure and agency debate
+After the functionalists and the structuralists (orthodoxy that collapsed), and after the critics from hermeneutics that revive the Verstehen/Erklären debate
+Double Hermeneutics and Teoría social informada hermenéuticamente
+Theory of structuration: theory of the subject, without falling into subjectivism. Both subject (human agent) and object (society, institutions) are made in and of recurrent practices.
+Describes a continuous process of action-monitoring, with unconscious components, and a conscience pratique, with a simplistic view of the cognitive, but there nonetheless.
+Structure is what permits something to be emitted, and what results from the emission (e.g. in language.), which is the way to escape a dualism in explaining the relation of part to totality.
+
+-->
+
+\add{steiner-autonomy-2009, who connects Giddens (and others) with the questions of social cognition}
+
+Anthropology and sociology have produced considerable amounts of work concerned with the nature of culture in relation to psychology, the most prominent parts of which are reviewed by @risjord_models_2012.
+The sample I exposed here represents the approaches which have most influenced the initial questions of the present thesis.
+Common behind the works of Mauss, Bourdieu, Giddens, and authors contemporary to each of them, lies a certain interest in eliminating biology-culture, nature-nurture, or substance-form dualisms which we routinely rely on in our conception of life.
+This concern has remained central in contemporary social anthropology, and is worth keeping in mind when studying the cultural evolution approach that I will focus on in what follows.
+
+While current writings often argue that such a separation is not "sharp" [@acerbi_cultural_2016 p. 2; relying on @morin_how_2016], the dualism it comes from permeates most of our theories about non-human animals, human beings, and life in general.
+This thesis is no exception, as I am myself an apprentice of the traditions I present and discuss here.
```

Comment:
Est-ce que passer ceci vers la fin de l'intro, according to option 1 ci-dessus, rendrait moins violent ? (À ce stade j'aurai bien introduit l'évolution biologique je crois)